src/nightly.js

```diff
@@ -45,19 +54,26 @@ module.exports = function*(argv) {
             desc: 'Don\'t actually publish to npm, just print what would be run.',
             type:'boolean'
         })
+        .options('ignore-test-failures', {
+            desc: 'Run the tests for cli and lib but don\'t fail the build if the tests are failing',
+            type:'boolean',
+            alias : 'ignoreTestFailures'
+        })
         .argv;
 
     if(argv.h) {
         optimist.showHelp();
         process.exit(1);
     }
-    
+
     //Grab currently published nightly version so we can unpublish it later
-    //Assumes lib and cli have same version
-    var oldNightlyVersion = yield executil.execHelper(executil.ARGS('npm view cordova dist-tags.nightly'));
-    console.log(oldNightlyVersion);
+    var oldNightlyVersions = {};
+    oldNightlyVersions.cli = yield executil.execHelper(executil.ARGS('npm view cordova dist-tags.nightly'));
+    oldNightlyVersions.lib = yield executil.execHelper(executil.ARGS('npm view cordova-lib dist-tags.nightly'));
+    console.log('oldNightlyVersions: ' + JSON.stringify(oldNightlyVersions));
```

Comment:
Please use `apputil.print` to make logging consistent. Also could you please update this to print human-readable output instead of raw JSON